test/groupBy.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+var S = require('..');
+
+var eq = require('./internal/eq');
+
+
+test('groupBy', function() {
+
+  eq(typeof S.groupBy, 'function');
+  eq(S.groupBy.length, 2);
+  eq(S.groupBy.toString(), 'groupBy :: (a -> a -> Boolean) -> Array a -> Array (Array a)');
+
+  var productsOf3 = S.curry2(function(a, b) {
+    return a * b % 3 === 0;
+  });
```

Comment:
We should avoid `S.curry2` here, as it will return a function which can be applied `(x)(y)`-style or `(x, y)`-style. Let's define a curried function manually:

```javascript
function productsOf3(x) {
  return function(y) {
    return x * y % 3 === 0;
  };
}
```